frontend/src/components/shared/Footer.js

```diff
@@ -1,7 +1,98 @@
+                            </Link>
+                            <Link href='/' underline="none">
+                                <Typography variant='subtitle1' color='primary.light'> General</Typography>
+                            </Link>
+                            <Link href='/' underline="none">
+                                <Typography variant='subtitle1' color='primary.light'> Projects</Typography>
+                            </Link>
+                        </Box>
+                        <Box sx={{ mx: 5 }}>
+                            <Typography variant='h6' color='primary.light' mb={2}> For Students </Typography>
+                            <Link href='/apply/students' underline="none">
+                                <Typography variant='subtitle1' color='primary.light'> About</Typography>
+                            </Link>
+                            <Link href='/apply/students' underline="none">
+                                <Typography variant='subtitle1' color='primary.light'> FAQ</Typography>
+                            </Link>
+                        </Box>
+                        <Box sx={{ mx: 5 }}>
+                            <Typography variant='h6' color='primary.light' mb={2}> For Nonprofits </Typography>
+                            <Link href='/apply/nonprofits' underline="none">
+                                <Typography variant='subtitle1' color='primary.light'> About</Typography>
+                            </Link>
+                            <Link href='/apply/nonprofits' underline="none">
+                                <Typography variant='subtitle1' color='primary.light'> FAQ</Typography>
+                            </Link>
+                        </Box>
+                        <Box sx={{ mx: 5 }}>
+                            <Typography variant='h6' color='primary.light' mb={2}> Contact Us </Typography>
+                            <Box sx={{ display: "flex", flexDirection: "row" }}>
+
+                                <Box sx={{ mt: 0.5 }}>
+                                    <AiOutlineMail style={{ color: "white", fontSize: "1.5em" }} size={15} />
+                                </Box>
+                                <Tooltip disableFocusListener disableTouchListener title="uofa-blueprint@gmail.com">
+                                    <Typography variant='subtitle1' color='primary.light' ml={1} > Email</Typography>
+                                </Tooltip>
+                            </Box >
+                            <Box sx={{ display: "flex", flexDirection: "row" }}>
+                                <Box sx={{ mt: 0.5 }}>
+                                    <AiOutlineLinkedin style={{ color: "white", fontSize: "1.5em" }} size={15} />
+                                </Box>
+                                <Link href="https://www.linkedin.com/company/uofa-blueprint/" target="_blank" underline="none" >
```

Comment:
can the Link and Tooltip wrap both the text and logo so the logo also do something if clicked/hovered